app/domain/authentication/authn_oidc/validate_status.rb

```diff
@@ -26,9 +26,10 @@ def validate_secrets
       end
 
       def fetch_oidc_secrets
-        @oidc_secrets ||= @fetch_oidc_secrets.(
+        @oidc_secrets ||= @fetch_authenticator_secrets.(
```

Comment:
Let's change this to "@oidc_authenticator_secrets" because that is how we have changed it in authn_oidc/authenticate.rb